ykcompile/src/lib.rs

```diff
@@ -607,6 +721,8 @@ impl<TT> TraceCompiler<TT> {
         // Jump to the label that reserves stack space for spilled locals.
         dynasm!(self.asm
             ; jmp ->reserve
+            ; ->crash:
```

Comment:
Did you (manually) check if the trace crashes ok?

Reply:
Yes, I changed one of the tests to overflows in the trace and it segfaulted with `Invalid operation` (or something similar).